cloud/deployment/deployment_test.go

```diff
@@ -813,17 +813,17 @@ func TestLogs(t *testing.T) {
 		// Mock GetDeploymentLogsWithResponse to return an error
 		mockCoreClient.On("GetDeploymentLogsWithResponse", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(&mockGetDeploymentLogsResponse, errMock).Once()
 
-		err := Logs(deploymentID, ws, "", false, false, false, logCount, mockPlatformCoreClient, mockCoreClient)
+		err := Logs(deploymentID, ws, "", "", false, false, false, logCount, mockPlatformCoreClient, mockCoreClient)
 		assert.ErrorIs(t, err, errMock)
 
 		mockPlatformCoreClient.AssertExpectations(t)
 		mockCoreClient.AssertExpectations(t)
 	})
 
 	t.Run("query for more than one log level error", func(t *testing.T) {
-		err := Logs(deploymentID, ws, "", true, true, true, logCount, mockPlatformCoreClient, mockCoreClient)
+		err := Logs(deploymentID, ws, "", "", true, true, true, logCount, mockPlatformCoreClient, mockCoreClient)
 		assert.Error(t, err)
-		assert.Contains(t, err.Error(), "cannot query for more than one log level at a time")
+		assert.Equal(t, err.Error(), "cannot query for more than one log level and or keyword at a time")
```

Comment:
```suggestion
		assert.Equal(t, err.Error(), "cannot query for more than one log level and/or keyword at a time")
```
To keep consistent if accepting the suggestion from other review comment.